scripts/artifacts-building/user_rcbops_artifacts_building.yml

```diff
@@ -0,0 +1,110 @@
+## Generic variables
+mirror_base_url: "http://rpc-repo.rackspace.com"
+apply_security_hardening: False #Makes gates faster!
+
+## PIP
+# openstack_repo_url is the address of the repo server, so
+# everything is fine if the repo has the staged artifacts.
+# However, for builds without repos (like building containers),
+# we override it with what we want, i.e. the rax mirror
+openstack_repo_url: "{{ mirror_base_url }}"
+
+# TODO(evrardjp): remove re-wire this variable with
+# rpc_release per appropriate branch/tag (read other files).
+pip_artifacts_version: "master"
+
+# TODO(evrardjp) remove this when openstack_release is wired with rpc_release
```

Comment:
This is actually already done, see https://github.com/rcbops/rpc-openstack/blob/master/rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml#L22